build-system/pr-check.js

```diff
@@ -200,6 +200,13 @@ const command = {
   testBuildSystem: function() {
     execOrDie('npm run ava');
   },
+  testDocumentLinks: function(files) {
+    files.forEach((file) => {
+      if (isDocFile(file)) {
+        execOrDie('build-system/test-links.sh ' + file);
```

Comment:
Could we do this in JS? 
- [ ] create a new file in `tasks` folder.
- [ ] Use https://github.com/tcort/markdown-link-check/blob/master/markdown-link-check as reference to read file, replace localhost and then call `markdownLinkCheck()` JS method instead of the binary.
- [ ] call the newly created task from here similar to other tasks like `compile`.

my experience with BASH has always involved some incompatibility between platforms, would be nice to avoid it.